fdbserver/DataDistribution.actor.cpp

```diff
@@ -4533,13 +4600,47 @@ ACTOR Future<Void> keyValueStoreTypeTracker(DDTeamCollection* self, TCServerInfo
 	    wait(brokenPromiseToNever(server->lastKnownInterface.getKeyValueStoreType.getReplyWithTaskID<KeyValueStoreType>(
 	        TaskPriority::DataDistribution)));
 	server->storeType = type;
-
 	if (type != self->configuration.storageServerStoreType) {
 		if (self->wrongStoreTypeRemover.isReady()) {
 			self->wrongStoreTypeRemover = removeWrongStoreType(self);
 			self->addActor.send(self->wrongStoreTypeRemover);
 		}
 	}
+	return Never();
+}
+
+// Read storage metadata from database; Error is catched by the caller
```

Comment:
a nit: "Read storage metadata from database; Error is catched by the caller" -> "Read storage metadata from database; Error(s) are expected to be handled by the caller"?